fastplotlib/layouts/_plot.py

```diff
@@ -117,7 +132,18 @@ def show(
             self.toolbar = ToolBar(self)
             self.toolbar.maintain_aspect_button.value = maintain_aspect
 
-        return VBox([self.canvas, self.toolbar.widget])
+        if not sidecar:
+            return VBox([self.canvas, self.toolbar.widget])
+
+        if self.sidecar is None:
+            # sidecar doesn't like unpacking when kwargs are `None`
```

Comment:
unpacking `None` has nothing to do with sidecar. You cannot unpack a `None` type, `**` is only for unpacking dictionaries :D